lib/features/family/features/recommendation/organisations/widgets/organisation_detail_bottomsheet.dart

```diff
@@ -105,8 +105,9 @@ class OrganisationDetailBottomSheet extends StatelessWidget {
                     if (!isDonateButtonActive) {
                       EmptyWalletBottomSheet.show(
                         context,
-                        () => context.pushNamed(
```

Comment:
Just to be sure, having `await context.read<CreateTransactionCubit>().fetchActiveProfileBalance();` before the navigate does not fix the problem?

Reply:
await context.read<CreateTransactionCubit>().fetchActiveProfileBalance(); calls the same endpoint as 
 await context.read<ProfilesCubit>().fetchAllProfiles();
 so yes it would fix the problem 99% of the time the same way

Reply:
but sometimes still there is a longer delay thats why there is also a retry within this widget. This widget specifically cannot take 0 as a max amount and throws a grey screen so it cannot be pushed without a fallback

Reply:
Clear, ty